projects/ngx-cookie-service/src/lib/cookie.service.ts

```diff
@@ -12,9 +12,9 @@ export class CookieService {
   private readonly documentIsAccessible: boolean;
 
   constructor(
-    @Inject(DOCUMENT) private document: Document,
+    @Inject(DOCUMENT) private document: any,
```

Comment:
?

Reply:
Angular team fixed this error with Ivy, so we can use `@Inject(DOCUMENT) private document: Document`

but for users who are using `ViewEngine` it broke the library, issues were caused by this #144  and #145

Reply:
Thank you for the explanation!